apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,9 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY_PATH + '?u=' + encodeURIComponent(opts.url.replace(/ /g, '%20')) +
```

Comment:
Done in https://github.com/code-dot-org/code-dot-org/pull/6349/commits/41169d1e778abe4e95012c5c5b695022e57b9f64